### ISSUE TITLE
Cannot update from 3.0.0 to 3.5.2 due 'ER_NO_SUCH_TABLE' error

### ISSUE THREAD

**Issue by zmactep** [TryGhost/Ghost#11597]
### Issue Summary

Cannot update from 3.0.0 to 3.5.2 due 'ER_NO_SUCH_TABLE' error during `ghost update`:
```
Message: show columns from `emails` - ER_NO_SUCH_TABLE: Table 'ghost_production.emails' doesn't exist
```

It is looks like database migration is missing.

### To Reproduce

Just run `ghost update`:
```
+ sudo systemctl is-active ghost_139-59-187-50
✔ Checking system Node.js version
✔ Ensuring user is not logged in as ghost user
✔ Checking if logged in user is directory owner
✔ Checking current folder permissions
✔ Checking folder permissions
✔ Checking file permissions
✔ Checking content folder ownership
✔ Checking memory availability
✔ Checking for available migrations
✔ Checking for latest Ghost version
✔ Downloading and updating Ghost to v3.5.2
+ sudo systemctl stop ghost_139-59-187-50
✔ Stopping Ghost
✔ Linking latest Ghost and recording versions
+ sudo systemctl start ghost_139-59-187-50
+ sudo systemctl stop ghost_139-59-187-50
✖ Restarting Ghost
A GhostError occurred.

Message: show columns from `emails` - ER_NO_SUCH_TABLE: Table 'ghost_production.emails' doesn't exist
Suggestion: journalctl -u ghost_139-59-187-50 -n 50

Debug Information:
    OS: Ubuntu, v16.04.3 LTS
    Node Version: v10.16.3
    Ghost Version: 3.5.2
    Ghost-CLI Version: 1.13.1
    Environment: production
    Command: 'ghost update'

Additional log info available in: /home/pavel/.ghost/logs/ghost-cli-debug-2020-02-14T11_18_40_726Z.log

Try running ghost doctor to check your system for known issues.

You can always refer to https://ghost.org/docs/api/ghost-cli/ for troubleshooting.



? Unable to upgrade Ghost from v3.0.0 to v3.5.2. Would you like to revert back to v3.0.0? Yes
+ sudo systemctl is-active ghost_139-59-187-50
+ sudo systemctl reset-failed ghost_139-59-187-50
✔ Checking system Node.js version
✔ Ensuring user is not logged in as ghost user
✔ Checking if logged in user is directory owner
✔ Checking current folder permissions
✔ Checking folder permissions
✔ Checking file permissions
✔ Checking content folder ownership
✔ Checking for available migrations
✔ Checking for latest Ghost version
ℹ Downloading and updating Ghost [skipped]
+ sudo /var/www/ghost/current/node_modules/.bin/knex-migrator-rollback --force --v 3.0.0 --mgpath /var/www/ghost/current
✔ Rolling back database migrations
✔ Linking latest Ghost and recording versions
+ sudo systemctl start ghost_139-59-187-50
✔ Restarting Ghost
ℹ Removing old Ghost versions [skipped]
```

Log file contains this:
```
Debug Information:
    OS: Ubuntu, v16.04.3 LTS
    Node Version: v10.16.3
    Ghost Version: 3.5.2
    Ghost-CLI Version: 1.13.1
    Environment: production
    Command: 'ghost update'
Message: show columns from `emails` - ER_NO_SUCH_TABLE: Table 'ghost_production.emails' doesn't exist
Suggestion: journalctl -u ghost_139-59-187-50 -n 50
Stack: Error: show columns from `emails` - ER_NO_SUCH_TABLE: Table 'ghost_production.emails' doesn't exist
    at Server.server.close (/usr/lib/node_modules/ghost-cli/lib/utils/port-polling.js:38:28)
    at Object.onceWrapper (events.js:286:20)
    at Server.emit (events.js:198:13)
    at emitCloseNT (net.js:1618:8)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```

### Technical details:

* Ghost Version: 3.5.2
* Node Version: v10.16.3
* Browser/OS: Ubuntu, v16.04.3 LTS
* Database: MySQL

**Comment by kevinansfield** [comment]
Hey @zmactep 👋 Please try upgrading again, this issue was resolved in Ghost 3.6.0 that was just released 😄

**Comment by naz** [comment]
For reference, this issue was resolved with https://github.com/TryGhost/Ghost/commit/354e946e6e95f5d11c594bfb4e7939e45bb42903
